assets/js/components/Loadpoints/Loadpoint.vue

```diff
@@ -269,7 +269,10 @@ export default defineComponent({
 			return !!this.vehicleName;
 		},
 		vehicleHasSoc() {
-			return this.vehicleKnown && !this.vehicle?.features?.includes("Offline");
+			return (
+				(this.vehicleKnown && this.vehicle?.features?.includes("Offline")) ||
+				this.vehicleNotReachable
+			);
```

Comment:
No, this change is to broad. The condition should only be added in the arrival tab. Otherwise limit selectors and planning UI will be in a mode that cannot be satisfied.